### DISCUSSION TITLE
mindsdb_sdk connection error

### DISCUSSION THREAD

**Discussion by Kyungmin-Yu** [mindsdb/mindsdb#9516] [Get help]
Hello.
I have issues in connection to local host using mindsdb_sdk. 
I am running on mac os, python 3.9.6 environment and installed mindsdb and mindsdb_sdk using pip install. 

When I run the code below, 'Connected to MindsDB running locally' is successfully printed. 
However, when I tried to run codes using the server object, connection error arises.

[code]
import mindsdb_sdk
# To connect to localhost and default port (47334)
server=mindsdb_sdk.connect()
print('Connected to MindsDB running locally')

server.databases.list()
print(server.status()['mindsdb_version'])

[error]
HTTPConnectionPool(host='127.0.0.1', port=47334): Max retries exceeded with url: /api/status (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108b79190>: Failed to establish a new connection: [Errno 61] Connection refused'))

**Comment by ZoranPandovski** [comment]
Can you access mindsdb directly without the SDK?
